### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by daxinnan** [PetrKryslUCSD/HowToUseJuliaWithSublimeText3#1]
Hi,there,
     If I use them on Ubuntu 18.04, do I need to install Terminal package? I'm the new to use.

**Comment by PetrKryslUCSD** [comment]
I haven't done this myself, but I believe that which packages need to be installed does not depend on the operating system. So, yes. Terminal is needed.

**Comment by dawbarton** [comment]
Personally I don't use the Terminal package and just use a separate shell (tmux in my case) since SendCode can deal with it directly. (IIRC SendCode can also work with the Gnome Terminal directly - the question really is do you want the terminal embedded in your Sublime window or are you happy with a separate floating window.)

**Comment by PetrKryslUCSD** [comment]
Is the tmux built-in into the main window, or separate? How is it set up,
if you would share that know-how?
Petr Krysl
Professor
University of California, San Diego
9500 Gilman Dr., #0085, La Jolla, CA 92093
Skype: Petr.Krysl.UCSD.EDU
<https://mailtrack.io/trace/link/3441810003af2d4cbe0eae6cf8f5d819ad2ba3fc?url=http%3A%2F%2FPetr.Krysl.UCSD.EDU&userId=1197778&signature=00cf8215ec900974>
http://hogwarts.ucsd.edu/~pkrysl/
<https://mailtrack.io/trace/link/e6d6693f52a5628fb86318ae1b89a3d46d569691?url=http%3A%2F%2Fhogwarts.ucsd.edu%2F~pkrysl%2F&userId=1197778&signature=227bfe6abe47882b>

**Comment by dawbarton** [comment]
I have tmux running in gnome-terminal; I've got a profile set up with the command `tmux new julia` (`julia` is on my path). The defaults for SendCode on linux are to send everything to tmux and so it works out the box.

SendCode (Linux).sublime-settings:
```
    "julia" : {
        "prog": "tmux",
        "bracketed_paste_mode": true
    },
```

![image](https://user-images.githubusercontent.com/1260244/64699173-1d1df500-d49c-11e9-9d41-a298fd125fb8.png)

On my Windows machine I use ConEmu as my terminal (since the default windows terminal is rubbish) and again, SendCode just works (though I did have to provide the right path to ConEmu in the config since the default one wasn't correct).

I did try the terminal package in Sublime but I never felt it worked as well as a dedicated terminal.
